pkg/volume/csi/nodeinfomanager/nodeinfomanager.go

```diff
@@ -612,28 +577,11 @@ func removeMaxAttachLimit(driverName string) nodeUpdateFunc {
 	}
 }
 
-// validateCSINodeInfo ensures members of CSINodeInfo object satisfies map and set semantics.
-// Before calling CSINodeInfoInterface.Update(), validateCSINodeInfo() should be invoked to
-// make sure the CSINodeInfo is compliant
-func validateCSINodeInfo(nodeInfo *csiv1alpha1.CSINodeInfo) error {
-	if len(nodeInfo.Status.Drivers) < 1 {
-		return fmt.Errorf("at least one Driver entry is required in driver statuses")
-	}
-	if len(nodeInfo.Spec.Drivers) < 1 {
-		return fmt.Errorf("at least one Driver entry is required in driver specs")
-	}
-	if len(nodeInfo.Status.Drivers) != len(nodeInfo.Spec.Drivers) {
-		return fmt.Errorf("")
-	}
-	// check for duplicate entries for the same driver in statuses
+// validateCSINode ensures members of CSINode object satisfies map and set semantics.
+// Before calling CSINodeInterface.Update(), validateCSINode() should be invoked to
+// make sure the CSINode is compliant
+func validateCSINode(nodeInfo *storage.CSINode) error {
```

Comment:
We don't need this anymore now that the validation is done by the api server

Reply:
Ok